test/test_dists.jl

```diff
@@ -169,6 +169,13 @@ end
         @inferred evaluate(Jaccard(), [1, 2, 3], [1, 2, 3])
         @inferred evaluate(Jaccard(), [true, false, true], [false, true, true])
 
+        # Test Bray-Curtis. Should be 1 if no elements are shared, 0 if all
+        # are the same
+        @test braycurtis([1,0,3],[0,1,0]) == 1.0
+        @test braycurtis(rand(10), zeros(10)) == 1.0
+        @test braycurtis([1,0],[1,0]) == 0.0
+
```

Comment:
I guess `Bool` case has to be tested as well.
Also, there's `T`, `_x`, `_y` loop above, I think it makes sense to add `braycurtis()` tests there as well (that should test `Float64` case).

Reply:
I added `braycurtis()` to the loop, as suggested, but I don't think Bray Curtis is defined for the boolean case, or at least it's not really an appropriate dissimilarity measure for presence/absence type situations.

Reply:
You have defined `eval_start()` for a `Bool` case, so I supposed it have to be tested.
E.g `Euclidean` or `Cityblock` also support boolean vectors.